### PR TITLE
Switch direct dependencies to rustls

### PR DESCRIPTION
**Pull request by mkpankov**
Work on #2072.

I will work on transitive dependencies and removal of native dependency from RPM as a follow-up.

Signed-off-by: Michael Pankov <work@michaelpankov.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2073)

### CODE REVIEW

File: iml-agent/src/main.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 use env::{MANAGER_URL, PEM};
+
```
Comment (johnsonw) [inline iml-agent/src/main.rs:6]: Doesn't seem necessary

Comment (mkpankov) [reply iml-agent/src/main.rs:6]: Done

File: iml-manager-cli/Cargo.toml [hunk 2]
```diff
@@ -21,8 +21,8 @@ indicatif = "0.14"
 lazy_static = "1.4"
 number-formatter = {path = "../number-formatter", version = "0.1"}
 prettytable-rs = "0.8"
-reqwest = {version = "0.10", features = ["default-tls", "json"]}
-serde = {version = "1", features = ["derive"]}
+reqwest = { version = "0.10", features = ["rustls-tls", "json"] }
+serde = { version = "1", features = ["derive"] }
```
Comment (jgrund) [inline iml-manager-cli/Cargo.toml:25]: Not sure where I'm getting it from, but my vscode has an autoformatter for .toml that removes the space between the braces.

Comment (mkpankov) [reply iml-manager-cli/Cargo.toml]: Done

**Comment by mkpankov** [comment]
@jgrund the tests have passed

**Pull request merged**